tests/projects/create.test.js

```diff
@@ -0,0 +1,279 @@
+        .send({ ...testProject, name: "" });
+      expect(res.statusCode).toBe(400);
+      expect(res.body.success).toBe(false);
+      expect(res.body).toHaveProperty("message");
+      expect(Array.isArray(res.body.errors)).toBe(true);
+      expect(res.body.errors).toContain("Name is required");
+    });
+
+    it("shouldn't create a project with too long name", async () => {
+      const res = await request(app)
+        .post("/projects")
+        .set("Authorization", `Bearer ${token}`)
+        .send({ ...testProject, name: "a".repeat(101) });
+      expect(res.statusCode).toBe(400);
+      expect(res.body.success).toBe(false);
+      expect(res.body).toHaveProperty("message");
+      expect(Array.isArray(res.body.errors)).toBe(true);
+      expect(res.body.errors).toContain("Name must be under 100 characters");
+    });
+
+    it("shouldn't create a project without description", async () => {
+      const res = await request(app)
+        .post("/projects")
+        .set("Authorization", `Bearer ${token}`)
+        .send({ ...testProject, description: "" });
+      expect(res.statusCode).toBe(400);
+      expect(res.body.success).toBe(false);
+      expect(res.body).toHaveProperty("message");
+      expect(Array.isArray(res.body.errors)).toBe(true);
+      expect(res.body.errors).toContain("Description is required");
+    });
+
+    it("shouldn't create a project with too long description", async () => {
+      const res = await request(app)
+        .post("/projects")
+        .set("Authorization", `Bearer ${token}`)
+        .send({ ...testProject, description: "a".repeat(501) });
+      expect(res.statusCode).toBe(400);
+      expect(res.body.success).toBe(false);
+      expect(res.body).toHaveProperty("message");
+      expect(Array.isArray(res.body.errors)).toBe(true);
+      expect(res.body.errors).toContain("Description must be under 500 characters");
+    });
+
+    it("shouldn't create a project without cause of death", async () => {
+      const res = await request(app)
+        .post("/projects")
+        .set("Authorization", `Bearer ${token}`)
+        .send({ ...testProject, causeOfDeath: "" });
+      expect(res.statusCode).toBe(400);
+      expect(res.body.success).toBe(false);
+      expect(res.body).toHaveProperty("message");
+      expect(Array.isArray(res.body.errors)).toBe(true);
+      expect(res.body.errors).toContain("Cause of death is required");
+    });
+
+    it("shouldn't create a project with too long cause of death", async () => {
+      const res = await request(app)
+        .post("/projects")
+        .set("Authorization", `Bearer ${token}`)
+        .send({ ...testProject, causeOfDeath: "a".repeat(101) });
+      expect(res.statusCode).toBe(400);
+      expect(res.body.success).toBe(false);
+      expect(res.body).toHaveProperty("message");
+      expect(Array.isArray(res.body.errors)).toBe(true);
+      expect(res.body.errors).toContain("Keep cause of death under 255 characters");
```

Comment:
The only thing that failed out of 81.
```bash
Expected value: "Keep cause of death under 255 characters"
Received array: ["Keep cause of death under 100 characters"]
```
```suggestion
      expect(res.body.errors).toContain("Keep cause of death under 100 characters");
```
or see suggestions for in projectSchema

Reply:
Ahh yeah I committed the bots correction but didn't update the test.